packages/jest-jasmine2/src/jasmine/createSpy.js

```diff
@@ -30,9 +30,9 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 */
 /* eslint-disable sort-keys */
 
-import CallTracker from './call_tracker';
+import CallTracker from './callTracker';
```

Comment:
I think the file name should be in pascal case – `CallTracker`. It's used as a class. Same for `SpyStrategy` below